R/tests.R

```diff
@@ -1,13 +1,3 @@
-
-renv_tests_program <- function(name) {
```

Comment:
Was only used by the download test; we have other more specific helpers for python.